pkg/scheduler/eventhandlers.go

```diff
@@ -468,6 +469,17 @@ func addAllEventHandlers(
 	)
 }
 
+func responsibleForCSI(sched *Scheduler) bool {
```

Comment:
In which case this would return false? It seems you listed all Filter name under `nodevolumelimits`.

Reply:
All these plugins does a same thing: Check if the number of volumes attached to each node exceeds the limit. `NodeVolumeLimits` is used for CSI volumes, `AzureDiskLimits`, `GCEPDLimits`, `EBSLimits`, and `CinderLimits` are used for legacy azure/gce/ebs/cinder volumes. This is common on cloud platforms, but in many on-premises environments, users use non cloud volumes, such as Ceph, which doesn't support a limit on attached volumes number. So all these plugins can be disabled to remove unnecessary predicates.